oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Commit.java

```diff
@@ -352,7 +352,11 @@ private void applyToDocumentStore(RevisionVector baseBranchRevision)
             JournalEntry doc = JOURNAL.newDocument(store);
             doc.modified(modifiedNodes);
             Revision r = revision.asBranchRevision();
-            store.create(JOURNAL, singletonList(doc.asUpdateOp(r)));
+            boolean success = store.create(JOURNAL, singletonList(doc.asUpdateOp(r)));
+            if (!success) {
+                LOG.error("Failed to update journal for revision {}", r);
+                LOG.debug("Failed to update journal for revision {} with doc {}", r, doc);
```

Comment:
```suggestion
                LOG.debug("Failed to update journal for revision {} with doc {}", r, doc.format());
```

IIUC JournalEntry.toString() only logs the id, so it would probably have to be something like `format()`

Reply:
ok, done; should I  merge?